website/docs/r/iam_group_policy_attachment.markdown

```diff
@@ -37,3 +37,9 @@ The following arguments are supported:
 
 * `group`  (Required) - The group the policy should be applied to
 * `policy_arn`  (Required) - The ARN of the policy you want to apply
+
+## Import
+ IAM group policy attachments can be imported using the group name and policy arn separated by `/`.
```

Comment:
Looks like there's an additional space in front of this line and the next. Will fix on merge. 👍